macros/sql/unpivot.sql

```diff
@@ -23,22 +27,26 @@ Arguments:
   {%- set cols = adapter.get_columns_in_relation(table) %}
 
   {%- for col in cols -%}
-    {%- if col.column.lower() not in exclude|map('lower') -%}
+    {%- if col.column.lower() not in remove|map('lower') and col.column.lower() not in exclude|map('lower') -%}
       {% set _ = include_cols.append(col) %}
     {%- endif %}
   {%- endfor %}
 
-  {%- for col in include_cols -%}
 
+  {%- for col in include_cols -%}
     select
       {%- for exclude_col in exclude %}
         {{ exclude_col }},
       {%- endfor %}
-      cast('{{ col.column }}' as {{ dbt_utils.type_string() }}) as field_name,
-      cast({{ col.column }} as {{ cast_to }}) as value
+
+      cast('{{ col.column }}' as {{ dbt_utils.type_string() }}) as "{{ field_name }}",
```

Comment:
```suggestion
      cast('{{ col.column }}' as {{ dbt_utils.type_string() }}) as {{ field_name }},
```

I don't think we necessarily want to add quotes here. This is sensible on Redshift/pg, but causes problems on BigQuery and Snowflake. In short:
- BigQuery uses backticks (not double quotes) to quote identifiers
- Snowflake treats unquoted identifiers as upper-case, so quoting a lower-case identifier constitutes a breaking change. More info on this (bizarre and upsetting) behavior here: https://docs.getdbt.com/docs/configuring-quoting#section-snowflake